src/main/java/com/accantosystems/stratoss/vnfmdriver/driver/GrantDriver.java

```diff
@@ -124,7 +139,10 @@ public Grant getGrant(String grantId) throws GrantRejectedException, GrantProvid
         final HttpEntity<Void> requestEntity = new HttpEntity<>(headers);
         final ResponseEntity<Grant> responseEntity;
         try {
+            UUID uuid = UUID.randomUUID();
+            LoggingUtils.logEnabledMDC(grantId, MessageType.REQUEST, MessageDirection.RECEIVED, uuid.toString(),MediaType.APPLICATION_JSON.toString(), "https",getRequestProtocolMetaData(url) ,null);
```

Comment:
driverRequestId can't be null. Can't we take the lcmOpOccId from Grant object and have the both request and response logs after making the external call?